infra/modules/azure_postgres_server/outputs.tf

```
@@ -0,0 +1,28 @@
+
+output "private_endpoint" {
+
+  value       = azurerm_private_endpoint.postgre_pe.id
+  description = "Private Endpoint resource ID for the PostgreSQL Flexible Server"
+}
+
+output "postgres" {
+  value = {
+    name = azurerm_postgresql_flexible_server.this.name
+    id   = azurerm_postgresql_flexible_server.this.id
```

We always return the resource group name as well

```suggestion
    id   = azurerm_postgresql_flexible_server.this.id
    resource_group_name = azurerm_postgresql_flexible_server.this.resource_group_name
```